### ISSUE TITLE
Customer cannot search out the subscription in the "Account--Subscripitons" page.

### ISSUE THREAD

**Issue by jiangweiyi** [servicecatalog/development#67]
Date: 2016/05/06 
[Test category]: creative Test

[Steps to reproduce]
1. Login as supplier, define a service, set price model, publishing option for it, then activate this service.
2. Login as customer, subscribe this service set its name as "test", put the configuration and price model information, then assign this subscription to himself.
3. Go to "My subscription" page to search this subscription as "test".
4. Go to "Account"->"Subscriptions" page to search this subscription as "test".

[Expected result]
After step 3 and 4, the subscription can be searched out.

[Observed result] 
After step3, it is searched out; after step4, it is not searched out.

**Comment by StavrevaS** [comment]
Duplicate of  [issue #76](https://github.com/servicecatalog/development/issues/76)

**Comment by jiangweiyi** [comment]
Verified by FNST
[verified-version] V16.0 fix5
[verified-details]Now the subscriptions can be searched out by customer in the "Account"--"Subscriptions" page, so this bug can be verified.
